.github/workflows/tests.yml

```diff
@@ -153,80 +154,47 @@ jobs:
     if: ${{github.event_name == 'pull_request' && fromJson(needs.setup_tests.outputs.syntax-only) == false}}
     strategy:
       matrix:
-        runner: ${{fromJson(needs.setup_tests.outputs.runners)}}
+        include:
+          - runner: '12-arm64'
+          - runner: '12'
+          - runner: '11-arm64'
+          - runner: '11'
+          - runner: '10.5'
```

Comment:
```suggestion
          - runner: '10.15'
```

Reply:
🤦‍♂️ and I was wondering when some Catalina runner will finally be available